features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2782,6 +2785,16 @@
                 <BasePath>/console/</BasePath>
             </Context>
         </WebApp>
+        <Servlet>
+        {% for servlet in tenant_context.rewrite.servlets %}
```

Comment:
Shouldn't tenant_context be organization_context? check other places as well
```suggestion
        {% for servlet in organization_context.rewrite.servlets %}
```

Reply:
Yes. fixed by - https://github.com/wso2/carbon-identity-framework/pull/4239/commits/6b320902fb5deffe07c447086292a2645a72b2c9